### ISSUE TITLE
Requirements Installing in directory not in path

### ISSUE THREAD

**Issue by maxwiesner** [aws/aws-mwaa-local-runner#115]
Wondering if anyone else has run into this issue. I only have one requirement in my `dags/requirements.txt` file which is `fastavro==1.4.9`. 

This is an issue that is getting reproduced in my MWAA instance as well. Here is the output that I get when I run the local runners test-requirements command. 

```bash
03:56:10 mymac airflow ±|master|→ ./mwaa-local-env test-requirements
Container amazon/mwaa-local:2.0.2 exists. Skipping build
grep: .env: No such file or directory
declare -x AIRFLOW_HOME="/usr/local/airflow"
declare -x HOME="/usr/local/airflow"
declare -x HOSTNAME="a196fa1cc653"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
declare -x PWD="/usr/local/airflow"
declare -x SHLVL="1"
declare -x TERM="xterm"
Installing requirements.txt
Collecting fastavro==1.4.9
  Downloading fastavro-1.4.9-cp37-cp37m-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (2.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 13.1 MB/s eta 0:00:00
Installing collected packages: fastavro
  WARNING: The script fastavro is installed in '/usr/local/airflow/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed fastavro-1.4.9
```

Is there a way to alter the path variable for MWAA even without being able to ssh into and do it on every resource (because its on fargate and we cant do that), or more preferably is there something I am missing to have the requirements download in the path that is already declared?

**Comment by shubhsboson** [comment]
This happens to me , but it's just a warning , not really an issue. Are you unable to use fastavro ?

**Comment by maxwiesner** [comment]
Correct I cant use it in my MWAA instance because its not in the path, heres the error. I found a botchy workaround where I just load the module from its absolute path, but would like to find a real solution, since this is a paid service and should really have some documentation on how to fix this/ why its happening. Heres the error message I get, I have added the location of the file in the path as well with a env variable plugin, but it still doesn't register it. 

<img width="1373" alt="Screen Shot 2022-04-05 at 8 44 40 AM" src="https://user-images.githubusercontent.com/70331375/161780371-db59d5f9-6f67-4dd6-93a9-5af2d93f2e53.png">
